org.eclipse.scanning.event/src/org/eclipse/scanning/event/ConsumerImpl.java

```diff
@@ -194,7 +198,7 @@ protected void terminate(KillBean kbean) {
 			} catch (InterruptedException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either re-interrupt this method or rethrow the "InterruptedException". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2142)